Clarified that LINSERT will return 0 if the key does not exist (#2289)
redis/redis-doc: commands/linsert.md

@return

> @integer-reply: the length of the list after the insert operation, or `-1` when
> the value `pivot` was not found, or `0` if the `key` does not exist.

Reviewer: ```suggestion
@integer-reply: the list length after a successful insert operation, `0` if the `key` doesn't exist, and `-1` when
the `pivot` wasn't found.
```

Author: thanks, it does look better
